src/generate.rs

```diff
@@ -182,17 +177,20 @@ impl<'a> Generate<Option<&'a DescribeState>> for DescribeState {
             }
         }
 
-        // Create subblocks from a full DescribeState
-        let subblocks = self.subblocks.clone().into_iter().map(|block| { block.generate(sp, cx, &self) }).collect();
-
         // Get a glob import of all items in scope to the module that `describe!` is called from.
         //
         // This glob is `pub use super::*` so that nested `describe!` blocks (which will also contain
         // this glob) will be able to see all the symbols.
-        let super_glob = cx.view_use_glob(sp, ast::Public, vec![cx.ident_of("super")]);
+        let super_glob = cx.item_use_glob(sp, ast::Public, vec![cx.ident_of("super")]);
+        let mut items = vec![super_glob];
+
+        // Create subblocks from a full DescribeState
+        for block in self.subblocks.clone().into_iter() {
```

Comment:
@reem done and rebased.

I also thought about `Some(super_glob).iter().chain(self.subblocks.iter()).map(.clone()...).collect()` but that would probably be a bit cryptic.. what do you think?